src/block/progress-circle/meta-data.json

```
@@ -0,0 +1,13 @@
+{
+	"name": "stackable/progress-circle",
+	"title": "Progress Circle",
+	"description": "Adds a circular progress bar block.",
+	"category": "common",
+	"usesContext": [ "postId", "postType", "queryId" ],
+	"keywords": [
+		"Progress",
+		"Progress Circle"
+	],
+	"textdomain": "stackable-ultimate-gutenberg-blocks",
+	"stk-type": "essential"
```

Better to move this to special because we want to keep the **essential** blocks to only the very basic ones
```suggestion
	"stk-type": "special"
```